packages/regl-worldview/src/types/index.js

```diff
@@ -174,11 +164,11 @@ export type Cube = BaseShape & {
 export type Cylinder = BaseShape;
 
 export type Line = BaseShape & {
-  points: (Point | Vec3)[],
+  points: Point[] | Vec3[],
 };
 
 export type PointType = BaseShape & {
-  points: (Point | Vec3)[],
+  points: (Point[] | Vec3)[],
```

Comment:
⚠️ Seems like a typo? Wasn't this supposed to be the same as Line? Bonus points: why didn't you get any Flow errors as a result?
```suggestion
  points: Point[] | Vec3[],
```

Reply:
We don't really have any usage of these components in tests, so we don't have any flow assertions. That's something I'll have to fix